jupyter/radon/radon.ipynb

```diff
@@ -0,0 +1,1995 @@
+    "As the number of regression predictors increases, the modeling choices increase.\n",
+    "\n",
+    "###  Modeling the regression intercept term\n",
+    "\n",
+    "A first multilevel version of the radon model extends the no-pooling model\n",
+    "by putting a distribution on the intercept term\n",
+    "\n",
+    "$\n",
+    "\\alpha_j \\sim \\mathrm{N}(\\mu_\\alpha,\\, {\\sigma_\\alpha}^2),\\ \\ \\ \\ \\mathrm{for}\\ \\ j\\, = 1, \\ldots, \\mathrm{J}\n",
+    "$\n",
+    "\n",
+    "In combination with the likelihood, the partial-pooling model is\n",
+    "\n",
+    "$\n",
+    "y_i = \\alpha_{j[i]} \\, + {\\beta}\\,x_i + {\\epsilon}_i \\\\\n",
+    "\\alpha_j \\sim \\mathrm{N}(\\mu_\\alpha,\\, {\\sigma_\\alpha}^2),\\ \\ \\ \\ \\mathrm{for}\\ \\ j\\, = 1, \\ldots, \\mathrm{J}\n",
+    "$\n",
+    "\n",
+    "All parameters are also estimated jointly by the model.\n",
+    "This model provides _partial pooling_ of information;\n",
+    "it pulls the estimates of $\\alpha_j$ towards the mean level $\\mu_\\alpha$, to a greater or lesser degree.\n",
+    "Partial pooling is a _soft constraint_ whose effect depends on the amount of group-level variance $\\sigma_\\alpha$.\n",
+    "As the variance increases, the amount of pooling decreases so that when\n",
+    "$\\sigma_\\alpha \\rightarrow \\inf$ there is no pooling; and when $\\sigma_\\alpha \\rightarrow 0$ there is complete-pooling.\n",
+    "\n",
+    "```stan\n",
+    "data {\n",
+    "  int<lower=1> N;  // observations\n",
+    "  int<lower=1> J;  // counties\n",
+    "  array[N] int<lower=1, upper=J> county;\n",
+    "  vector[N] x;\n",
+    "  vector[N] y;\n",
+    "}\n",
+    "parameters {\n",
+    "  real mu_alpha;\n",
+    "  real<lower=0> sigma_alpha;\n",
+    "  vector<offset=mu_alpha, multiplier=sigma_alpha>[J] alpha;  // non-centered parameterization\n",
+    "  real beta;\n",
+    "  real<lower=0> sigma;\n",
+    "}\n",
+    "model {\n",
+    "  y ~ normal(alpha[county] + beta * x, sigma);  \n",
+    "  alpha ~ normal(mu_alpha, sigma_alpha); // partial-pooling\n",
+    "  beta ~ normal(0, 10);\n",
+    "  sigma ~ normal(0, 10);\n",
+    "  mu_alpha ~ normal(0, 10);\n",
+    "  sigma_alpha ~ normal(0, 10);\n",
+    "}\n",
+    "generated quantities {\n",
+    "  array[N] real y_rep = normal_rng(alpha[county] + beta * x, sigma);\n",
+    "}\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3be9bf81",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "partial_pooling_alpha_model = CmdStanModel(stan_file=os.path.join('stan', 'radon_pp_alpha.stan'))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "92eac02d",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "partial_pooling_alpha_fit = partial_pooling_alpha_model.sample(data=radon_data)\n",
+    "partial_pooling_alpha_fit.summary().round(2)[:10]"
```

Comment:
```suggestion
    "partial_pooling_alpha_fit.summary().round(2).head()"
```


There are also a bunch of places in the appendices where you wrote `[:3]` style, but I think that is fine there.